docs/performance/quantization.md

```diff
@@ -28,7 +28,7 @@ Quantization in ONNX Runtime refers to 8 bit linear quantization of an ONNX mode
 
  For symmetric quantization:
  ```
- scale = abs(data_range_max, data_range_min) * 2 / (quantization_range_max - quantization_range_min)
+ scale = max(data_range_max, data_range_min) * 2 / (quantization_range_max - quantization_range_min)
```

Comment:
Thanks! Could you please change it to max(abs(data_range_max), abs(data_range_min)) to cover the case both *_max and *_min are negative?

Reply:
Ok, I will make a new PR : )